developer/src/tike/actions/dmActionsMain.pas

```diff
@@ -1047,9 +1047,9 @@ procedure TmodActionsMain.actWindowPrevUpdate(Sender: TObject);
 procedure TmodActionsMain.DataModuleCreate(Sender: TObject);
 begin
   actFileOpen.Dialog.Filter :=
-    'Keyman source files|*.kmn;*.kps;*.txt;*.bmp;*.ico;*.kpj;*.kvk;*.kvks;*.model.ts;*.tsv|'+
+    'Keyman source files|*.kmn;*.kps;*.txt;*.bmp;*.ico;*.kpj;*.kvk;*.kvks;*.model.ts;*.tsv;*.xml|'+
     'Projects files (*.kpj)|*.kpj|'+
-    'Keyboard files (*.kmn)|*.kmn|'+
+    'Keyboard files (*.kmn, *.xml)|*.kmn;*.xml|'+
```

Comment:
Yeah, this is technically part of a separate PR -- supporting loading .xml keyboard files -- can move it out if you want.